File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.py
from collections import Counter
class Solution:
    def minimumLength(self, s: str) -> int:
        c = Counter(s)
        ans = 0
        for key in c:
            v = c[key]
            if v % 2 == 0:
                ans += 2
            else:
                ans += 1
        return ans

File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        Map<Character, Integer> hm = new HashMap<>();
        for(int i = 0; i < s.length(); i++) {
            Character ch = s.charAt(i);
            int val = hm.getOrDefault(ch, 0);
            hm.put(ch, val + 1);
        }
        int ans = 0;
        for(Character k : hm.keySet()) {
            if (hm.get(k) % 2 == 0) {
                ans += 2;
            } else {
                ans += 1;
            }
        }
        return ans;
    }
}